src/AAFactory.sol

```diff
@@ -31,6 +31,18 @@ contract AAFactory {
     beacon = _beacon;
   }
 
+  function getBeaconProxyBytecodeHash() external view returns (bytes32) {
+    return beaconProxyBytecodeHash;
+  }
+
+  function getBeacon() external view returns (address) {
+    return beacon;
+  }
+
+  function getEncodedBeacon() external view returns (bytes memory) {
```

Comment:
just make the variables public, and solidity will create getters for you

Reply:
I'm keeping this one to do the encoding within solidity (instead of having do the encoding correctly within js), but I made the other two public.

Reply:
This is used from the front-end of the extension